src/utils/url.ts

```diff
@@ -105,6 +105,21 @@ export const isValidUrl = (url: string): boolean => {
   }
 };
 
+export const isStringAValidUrl = (urlString: string) => {
```

Comment:
why have you created a new function while there is `isValidUrl`? can you refactor the existing one?

Reply:
refactored d04b6de